app/src/main/res/layout/activity_maps.xml

```diff
@@ -15,7 +15,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
 		tools:context=".OutdoorMaps.View.MapsActivity"
- 	map:cameraZoom="16"
+ 	    map:cameraZoom="16"
```

Comment:
indented

Reply:
Its github that changes the layout of the code, weird, I know.